applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -842,6 +843,72 @@ static struct bt_conn_cb conn_callbacks = {
 
 };
 
+/**
+ * @brief	Callback handler for play/pause.
+ *
+ * @note	This callback is called from MCS after receiving a
+ *		command from the client or the local media player.
+ *
+ * @param[in]	play  Boolean to indicate if the stream should be resumed or paused.
+ */
+static void le_audio_play_pause_cb(bool play)
+{
+	int ret;
+
+	LOG_DBG("Play/pause cb, state: %d", play);
+
+	if (play) {
+		playing_state = true;
+
+		if (audio_streams[AUDIO_CH_L].ep->status.state ==
+		    BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+			ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_L],
+						     lc3_preset_nrf5340.codec.meta,
+						     lc3_preset_nrf5340.codec.meta_count);
+
+			if (ret) {
+				LOG_WRN("Failed to enable left stream");
+			}
+		}
+
+#if !CONFIG_STREAM_BIDIRECTIONAL
```

Comment:
I get why this is here, but it looks strange that bidir influences the right channel. Discuss if we can find another solution.

Reply:
I think this part will be refactored later when we introduce TWS with MIC return.

Reply:
Correct, it's because we currently only allow one headset and we decided it should be left device